---
permalink: /
title: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

I am a Bioinformatics Scientist specializing in Next-Generation Sequencing (NGS) data analysis and systems biology. My expertise lies in designing and implementing end-to-end bioinformatics pipelines for NGS data analysis across multiple projects. I have worked extensively in interdisciplinary environments, integrating multi-omics data, network-based approaches, and mathematical modeling to understand disease mechanisms and identify drug targets.

Currently, I am a Project Scientist II at CSIR-National Chemical Laboratory (NCL), where I initiated and developed research projects to investigate the role of GATA3 in breast cancer. I leveraged ChIP-seq, ATAC-seq, and regulatory gene network analysis to understand differential binding and signaling in luminal breast cancer. 

Previously, I led a fully remote team at [NonExomics](https://nonexomics.com/), an early-stage biotech startup, to deliver an on-time analysis of proprietary WGS and RNA-seq data. Our work contributed to the identification and experimental validation of novel cancer biomarkers.

Before that, during my postdoctoral research at IISER-Pune, I developed a proteogenomic pipeline to explore the role of non-canonical peptides in different systems. In one project, I conducted a pan-cancer analysis of publicly available RNA-seq and Ribo-seq datasets, characterizing novel peptides and assessing their potential as diagnostic and therapeutic targets. This included functional annotation, structure prediction, disease-associated mutation mapping, and in-silico drug screening.
	In another project, I evaluated state-of-the-art RNA-seq tools and developed an optimized comparative transcriptomics pipeline for cichlid fish species, shedding light on the emergence of protein-coding genes from noncoding regions and their role in speciation.

During my Ph.D. at [CSIR-National Chemical Laboatory](https://sites.google.com/site/biosystemsanalysis/Home), I collaborated closely with three experimental labs, serving as the primary bioinformatics lead. I developed computational workflows, analyzed high-throughput data, and built mathematical models to study human skin pigmentation.

I leveraged server-based computing, and AWS cloud computing services and extensively used R, Bash, and Python for data analysis, visualization, and workflow automation.

I am passionate about applying my expertise to elucidate disease mechanisms and drive translational research.
